build.gradle

```diff
@@ -114,7 +120,7 @@ publishing {
 
 // Custom task to ensure we can conditionally publish either a release or snapshot artifact
 // based on a command line switch. See github workflow files for more details on usage.
-task checkVersion {
+tasks.register('checkVersion') {
```

Comment:
IntelliJ said `tasks.register()` and later on `configureEach` below is more 1337

Reply:
h4x0r